examples/color-legend/example.json

```diff
@@ -0,0 +1,10 @@
+{
+  "path": "color-legend",
+  "title": "Color legends",
+  "exampleCss": ["main.css"],
```

Comment:
For examples with empty css files, we really should set `exampleCss: []` and shouldn't include a blank css file.